site/static/main.ts

```diff
@@ -29,8 +29,10 @@ function renderExample($target: Selection<any, any, any, any>, text: string) {
 
   const vis = $target.append('div').attr('class', 'example-vis');
 
+  // Decrease visual noise by removing $schema and description from code examples.
+  const textClean = text.replace(/(\s)+\"(\$schema|description)\": \".*?\",/g, '');
```

Comment:
Want to call out this change I made. I found the `$schema` and `description` properties to add unnecessary noise as the code blocks are fairly narrow. Can revert if we'd rather keep them.